doc/reference/reference_rock/vshard/vshard_ref.rst

```diff
@@ -641,6 +655,43 @@ Router public API
 
     Force wakeup of the bucket discovery fiber.
 
+.. _router_api-discovery_set:
+
+.. function:: vshard.router.discovery_set(mode)
+
+    Turn on/off the background discovery fiber used by the router to
+    find buckets.
+
+    :param mode: working mode of a discovery fiber. There are three modes: ``on``,
+                 ``off`` and ``once``
+
+    When the mode is ``on`` (default), the discovery fiber works all the lifetime
+    of the router. Even after all buckets are discovered, it will
+    still come to storages and download their buckets with some big
```

Comment:
Would be nice to clarify the rules for setting the period. There'll be questions.

Reply:
fixed in c826865